docs/en-US/component/space.md

```diff
@@ -119,22 +119,24 @@ space/fill-ratio
-| class      | Classname                       | string / Array / Object            | -                                                                           | -          |
-| direction  | Placement direction             | string                             | vertical/horizontal                                                         | horizontal |
-| prefixCls  | Prefix for space-items          | string                             | el-space                                                                    | -          |
-| style      | Extra style rules               | string / Array / Object            | -                                                                           | -          |
-| spacer     | Spacer                          | string / number / VNode            | -                                                                           | -          |
-| size       | Spacing size                    | string / number / [number, number] | -                                                                           | 'small'    |
-| wrap       | Auto wrapping                   | boolean                            | true / false                                                                | false      |
-| fill       | Whether to fill the container   | boolean                            | true / false                                                                | false      |
-| fill-ratio | Ratio of fill                   | number                             | -                                                                           | 100        |
-
-## Space Slot
+## API
+
+### Attributes
+
+| Name       | Description                     | Type                                                                                                                            | Default    |
+| ---------- | ------------------------------- | ------------------------------------------------------------------------------------------------------------------------------- | ---------- |
+| alignment  | Controls the alignment of items | ^[string]`'center' \| 'normal' \| 'stretch' \| ...` [align-items](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items) | center     |
+| class      | Classname                       | ^[string] / ^[object]                                                                                                           | -          |
+| direction  | Placement direction             | ^[string]`'vertical' \| 'horizontal'`                                                                                           | horizontal |
+| prefixCls  | Prefix for space-items          | ^[string]                                                                                                                       | -          |
+| style      | Extra style rules               | ^[string] / ^[object]`CSSProperties \| CSSProperties[] \| string[]`                                                             | -          |
```

Comment:
^[object]`CSSProperties \| CSSProperties[] \| string[]`

Does this type seem reasonable?

Reply:
At present, arrays and objects are unified into object types, and there is no good subdivision scheme.

Reply:
```
^[object] / ^[array]`CSSProperties[] \| string[]`
```
How about this?